app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -109,3 +118,139 @@ define [
+      if elem.data("publish")==true || elem.data("protect")==true
+        elem.removeClass(if elem.data("publish") then 'icon-publish' else 'icon-lock')
+        elem.addClass(if elem.data("publish") then 'icon-unpublish unpublish_recording_link' else 'icon-unlock unprotect_recording_link')
+        elem.text(if elem.data("publish") then 'Unpublish' else 'Unprotect')
+      else if elem.data("publish")==false || elem.data("protect")==false
+        elem.removeClass(if elem.data("publish")==false then 'icon-unpublish' else 'icon-unlock')
+        elem.addClass(if elem.data("publish")==false then 'icon-publish publish_recording_link' else 'icon-lock protect_recording_link')
+        elem.text(if elem.data("publish")==false then 'Publish' else 'Protect')
+
+    mouse_leave: (e) ->
+      elem = $(e.currentTarget)
+      if elem.data("publish")==true || elem.data("protect")==true
+        elem.removeClass(if elem.data("publish") then 'icon-unpublish unpublish_recording_link' else 'icon-unlock unprotect_recording_link')
+        elem.addClass(if elem.data("publish") then 'icon-publish' else 'icon-lock')
+        elem.text(if elem.data("publish") then 'Published' else 'Protected')
+      else if elem.data("publish")==false || elem.data("protect")==false
+        elem.removeClass(if elem.data("publish")==false then 'icon-publish publish_recording_link' else 'icon-lock protect_recording_link')
+        elem.addClass(if elem.data("publish")==false then 'icon-unpublish' else 'icon-unlock')
+        elem.text(if elem.data("publish")==false then 'Unpublished' else 'Unprotected')
+
+    ## BBB FRONT END FUNCTIONS FOR PUBLISH, PROTECT AND DELETE RECORDINGS
+    displaySpinner: (elem, publish, protect) ->
+      elem.prev('img.loader').show()
+      elem.remove()
+
+    togglePublishOrProtectButton: (parent, publish, protect) ->
+      spinner = parent.find('img.loader[data-action="'+ if publish then 'publish' else if protect then 'protect' else '' +'"]')
+      elem =  if publish == "true"
+                class: 'btn btn-small publish icon-publish'
+                text: I18n.t('conferences.recordings.published', 'Published')
+                publish: 'true'
+              else if publish == "false"
+                class: 'btn btn-small publish icon-unpublish'
+                text: I18n.t('conferences.recordings.unpublished', 'Unpublished')
+                publish: 'false'
+              else if protect=="true"
+                class: 'btn btn-small protect icon-lock'
+                text: I18n.t('conferences.recordings.protected', 'Protected')
+                protect: 'true'
+              else if protect=="false"
+                class: 'btn btn-small protect icon-unlock'
+                text: I18n.t('conferences.recordings.unprotected', 'Unprotected')
+                protect: 'false'
+      spinner.hide()
+      if publish
+        $('<a class="'+elem.class+'" data-publish="'+elem.publish+'">'+elem.text+'</a>').insertAfter(spinner)
```

Comment:
So `xsslint` is not particularly smart, it evaluates each expression pretty much in isolation, and it doesn't read the previous lines to see where stuff comes from. For hardcoded stuff like `class` and `data`, an alternative approach would be to name them differently ... `xsslint` assumes anything named `*Html` is safe, so you could rename those properties to `classHtml` and `dataHtml` respectively

As for the require, you meed to put `str/htmlEscape` higher up, right after `jst/conferences/newConference` ... the elements in that array correspond to the arguments immediately following it (`i18n!conferences` -> `I18n`, `jquery` -> `$`, etc.)